### PR TITLE
Swedish translation

### PR DESCRIPTION
**Pull request by Feilin-Ruidian**
Please see the comments (with "//" on some lines)

### CODE REVIEW

**Comment by jfresco** [comment]
Hi @Feilin-Ruidian. Please check that you are submitting an empty `en.json` file.
To gather context about where a key is being used, could be helpful to do a full text search with `t('<key name>')` and see in what view(s) is being used.
Also you can use the english version of the app and find the key values there.

**Comment by Feilin-Ruidian** [comment]
Hmm, that's weird. I was trying to submit a Swedish translation. Is that
visible at all?

2015-01-30 17:28 GMT+01:00 José Fresco <notifications@github.com>:

**Comment by jfresco** [comment]
The swedish translation is visible, but your Pull Request includes an empty `en.json` (english translation), which is not right ([see](https://github.com/DemocracyOS/app/pull/600/files)).

File: lib/translations/lib/se.json [hunk 1]
```diff
@@ -0,0 +1,349 @@
...
+  "common.or": "eller",
+  "common.ok": "OK",
+  "common.cancel": "Avbryt",
+  "common.back": "Bakåt",
+  "common.no-user-for-email" : "Ingen använder för nuvarande den email-adressen",
+
+  "DemocracyOS": "DemocracyOS",
+  "header.logo": "DemocracyOS",
+  "header.signin": "Logga in",
+  "Login to": "Logga in på", // Verify context
```
Comment (gvilarino) [inline lib/translations/lib/se.json]: Please don't add this stuff to `JSON` files, there shouldn't be comments in the language file.

**Comment by gvilarino** [comment]
@Feilin-Ruidian you just need to fix it in your `patch-1` branch and push it. The PR will update itself after that.

**Comment by gvilarino** [comment]
@Feilin-Ruidian I'll handle this, no worries. Thanks for the help!

**Comment by gvilarino** [comment]
The contribution was finally added properly by 7a9c1b817203ce1700a91d0f7d6d6b266c661e78

**Pull request closed without merging**